src/index.html

```diff
@@ -6,9 +6,56 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&family=Open+Sans:wght@400;500&family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" >
+        <img src="images/logo.png" alt="logo"
+        class="logo">
+      </a>
+      <nav class="navbar">
+        <ul class="navbar-list">
+          <li class="nav-item">
+            <a href="#" class="nav-link is-active">Apple</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="nav-link">Smartphones</a>
+          </li>
+
+          <li class="nav-item">
+            <a
+            href="#"
+            data-qa="hover"
+            class="nav-link">Laptops & Computers</a>
```

Comment:
follow the code style

```suggestion
            <a
              href="#"
              data-qa="hover"
              class="nav-link"
            >
              Laptops & Computers
            </a>
```